pubspec.yaml

```diff
@@ -1,12 +1,13 @@
 name: observable
-version: 0.21.0+1-dev
+version: 0.21.1-dev
 author: Dart Team <misc@dartlang.org>
 description: Support for marking objects as observable
 homepage: https://github.com/dart-lang/observable
 environment:
-  sdk: '>=1.23.0 <2.0.0'
+  sdk: '>=2.0.0-dev.12.0 <2.0.0'
```

Comment:
Should we bump this min version here too if it is impossible to version solve if you are running 2.0.0-dev.12.0?

Reply:
It *probably* doesn't matter, but sure.